test/types/bool/kushal/reportLiteralSize.chpl

```diff
@@ -0,0 +1,14 @@
+extern proc sizeof(e): size_t;
+
+/* Testing that the size reported is correct */
+writeln(sizeof(true:bool(8)));
+writeln(sizeof(false:bool(16)));
+writeln(sizeof(true:bool(32)));
+writeln(sizeof(false:bool(64)));
+
+writeln();
+/* Testing the that output shown is still the correct one */
+writeln(true:bool(8));
+writeln(false:bool(16));
+writeln(true:bool(32));
+writeln(false:bool(64));
```

Comment:
Not at all required, but a cute way to write this test would be to use a generic procedure:

    proc testbool(type t) {
      writeln(sizeof(true:t));
      writeln(sizeof(false:t));
      writeln(true:t);
      writeln(false:t);
    }
    testbool(bool(8));
    testbool(bool(16));
    etc.